api/integrations/lead_tracking/hubspot/tasks.py

```diff
@@ -27,41 +26,37 @@ def track_hubspot_lead(user_id: int, organisation_id: int = None) -> None:  # ty
             id=organisation_id
         )
 
-    hubspot_lead_tracker.create_lead(**create_lead_kwargs)
+    hubspot_lead_tracker.create_organisation_lead(**create_lead_kwargs)
 
 
 @register_task_handler()
-def update_hubspot_active_subscription(subscription_id: int) -> None:
+def track_hubspot_user_contact(user_id: int) -> None:
     assert settings.ENABLE_HUBSPOT_LEAD_TRACKING
 
-    from organisations.models import Subscription
+    from users.models import FFAdminUser
 
     from .lead_tracker import HubspotLeadTracker
 
-    subscription = Subscription.objects.get(id=subscription_id)
+    user = FFAdminUser.objects.get(id=user_id)
+    if not HubspotLeadTracker.should_track(user):
+        return
+
     hubspot_lead_tracker = HubspotLeadTracker()
-    hubspot_lead_tracker.update_company_active_subscription(subscription)
+
+    hubspot_lead_tracker.create_user_hubspot_contact(user)
 
 
 @register_task_handler()
-def track_hubspot_lead_without_organisation(user_id: int) -> None:
-    """
-    The Hubspot logic relies on users joining or creating an organisation
-    to be tracked. This should cover most use cases, but for users that
-    sign up but don't join or create an organisation we still want to be
-    able to track them.
-    """
+def update_hubspot_active_subscription(subscription_id: int) -> None:
+    from organisations.models import Subscription
 
-    from users.models import FFAdminUser
+    from .lead_tracker import HubspotLeadTracker
 
-    user = FFAdminUser.objects.get(id=user_id)
-    if hasattr(user, "hubspot_lead"):
-        # Since this task is designed to be delayed, there's a chance
-        # that the user will have joined an organisation and thus been
-        # tracked in hubspot already. If so, do nothing.
-        return
+    assert settings.ENABLE_HUBSPOT_LEAD_TRACKING
```

Comment:
The git diff is a bit odd here, I think just because we've moved the `assert` statement below the imports. Was there a reason for doing this?